orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/basetest/HandlerIntegrationTest.java

```diff
@@ -270,7 +270,7 @@ public IntegrationTestConfigurationService(
         }
 
         @Override
-        public String getTokenSigningKeyAlias() {
+        public String getIdTokenSigningKeyAlias() {
             return tokenSigningKey.getKeyAlias();
```

Comment:
If this is the ID token, should `tokenSigningKey` (and all the other bits that leads to...) also be named idToken?